components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/UpdateClaimConfiguration.java

```diff
@@ -131,6 +133,14 @@ private void updateSubjectClaimConfigs(SubjectConfig subjectApiModel, ServicePro
 
         if (subjectApiModel != null) {
 
+            if (subjectApiModel.getMappedLocalSubjectMandatory() != null &&
+                    subjectApiModel.getMappedLocalSubjectMandatory() &&
+                    subjectApiModel.getUseMappedLocalSubject() != null &&
+                    !subjectApiModel.getUseMappedLocalSubject()) {
```

Comment:
can simplify with Boolean.TRUE.equals() and Boolean.FALSE.equals()

Reply:
addressed with https://github.com/wso2/identity-api-server/pull/521/commits/c55442cff0175c0b4007f2803256c71b8bcaea36